lib/tropic/tropic2form.flow

```diff
@@ -1281,7 +1281,9 @@ tropic2Acc(b : Tropic, parent : TParentInfo, sheet : Stylesheet, metricsOnly : b
 					xscale = fdivide(ba2Metrics.width, ba1.metrics.width);
 					yscale = fdivide(ba2Metrics.height, ba1.metrics.height);
 					customScale = extractStruct(style, TFillScale(const(0.0))).scale;
-					scale = fmax3(xscale, yscale, customScale);
+					scale = 
```

Comment:
@NikitaFil please check this too.

Reply:
Don't change TFillScale behaviour, it will break existing content